common/changes/office-ui-fabric-react/ShimmerPerformance_2019-03-14-20-51.json

```diff
@@ -0,0 +1,9 @@
+{
+  "changes": [{
+    "packageName": "office-ui-fabric-react",
+    "comment": "Performance improvement of the shimmer component",
+    "type": "major"
```

Comment:
No major changes allowed as a separate PR. We have some strict procedures for that. With that said, we have a branch for all the breaking changes we want to include with the next major release.

Reply:
I removed this file for fabric-7